### PR TITLE
Sass workflow

### PR DESCRIPTION
**Pull request by paulbalandan**
**Description**
Follow up to #5715 

Adds a workflow to test whether changes to SCSS files are considered in the CSS, and vice versa.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [x] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by paulbalandan** [comment]
I'm not sure why the new workflow isn't running.

File: .github/workflows/test-scss.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - '**.scss'
+      - '**.css'
+      - '.github/workflows/test-scss.yml'
+
+jobs:
+  build:
+    name: Compilation of SCSS (Dart Sass)
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
```
Comment (kenjis) [inline .github/workflows/test-scss.yml:27]: Remove this.

**Comment by paulbalandan** [comment]
It seems `git diff-index` is not the way to go. I'll check for alternatives.

**Review by kenjis — APPROVED** [review]
LGTM

**Comment by lonnieezell** [comment]
Nice one. Thanks!

**Pull request merged**